CLA.md

```diff
@@ -0,0 +1,66 @@
+# `calibr8` CLA
+
+Thank you for your interest in contributing to Forschungszentrum Jülich GmbH - Institute for Bio- and Geosciences: IBG-1's `calibr8` project ("We" or "Us").
+
+The purpose of this contributor agreement is to clarify and document the rights granted by contributors to Us.
+To make this document effective, please follow the instructions below.
+
+# How to use this CLA
+
+If You are an employee and have created the Contribution as part of your employment, You need to have Your employers approval as a Legal Entity.
```

Comment:
I find capitalisation super weird in this text. Why are Contribution and Copyright capitalised, but employer is not?

Reply:
lawyers.. (German lawyers..)

Legal texts often have weird capitalization, and since this is the text we got from R-R we shouldn't change it. (Other than the project names of course.)